features/clip-path-boxes.yml

```diff
@@ -0,0 +1,8 @@
+name: Clip path boxes
+description: The `fill-box`, `stroke-box`, and `view-box` values for the `clip-path` set the part of the shape that will be used to clip the element.
```

Comment:
```suggestion
description: The `fill-box`, `stroke-box`, and `view-box` values for `clip-path` set an edge of the elements box to use as the clipping shape.
```

Reply:
I added an apostrophe in `element's` but wanted to verify that was the intent.

Reply:
the intent was possessive, but I'm dyslexic. The element has a box. :)

Reply:
I just wanted to confirm it wasn't like `attorneys general` or something 😆